src/form.ts

```diff
@@ -42,13 +49,13 @@ export class Form {
    */
   constructor(
     configuration: { form: FormConfig; widget: WidgetConfig },
-    onSubmitEvent?: (data: Record<string, FormDataEntryValue>) => void
+    onSubmitEvent?: (data: Record<string, FormDataEntryValue>) => Promise<void>
   ) {
     this.fullFormConfiguration = configuration.form;
     this.widgetConfiguration = configuration.widget;
     this.container = make('div', classes.container);
     this.form = this.createOpenForm(onSubmitEvent);
-    this.createMinimizedForm();
+    this.formCollapsed = this.createMinimizedForm();
```

Comment:
Method uses `minimized`, property uses `collapser`. I'd suggest to go with unified name